.azure-pipelines/baseline_test/baseline.test.template.yml

```diff
@@ -8,79 +8,77 @@ jobs:
     steps:
       - template: ../impacted_area_testing/get-impacted-area.yml
 
-  - job: run_impacted_area_test
+  - job: impacted_area_kvmtest
+    displayName: " "
```

Comment:
I think you don't need to set it empty here.

Reply:
This yml is using job matrix, then the job name would be `displayName + blank space + matrix`, and the blank space can't be removed. To make the baseline job name same as PR test, need to set displayName to " " and matrix = PR test job name